test/pageobjects/login.page.js

```diff
@@ -34,9 +102,119 @@ class LoginPage extends Page {
     /**
      * overwrite specific options to adapt it to page object
      */
-    open () {
-        return super.open('login');
+    open() {
```

Comment:
This should be another file and class, because this is a new page object, right ?

Reply:
I didn't understood this.
i need to create a different class in which I would write getters for that class, or should I write new file for getters, and new file for class?

Every time we move from one page to another that is another class, and another file we call something.page.js?
And, is it a rule - when we create new page, that page should have test of their own?
e.g. Lets say we have login page (easiest example)
Should I create login.page.js where I write getters and methods for this page, and login.spec where I can write these tests to execute?
Is there other files?

Reply:
LoginPage.js (file name), consists of class LoginPage, which extends BasePage. Class LoginPage contains getters (locators) and setters (methods that operate upon locator, clickOnSomething, SetSomething, etc) in it.

For every new page object, same implementation.

Reply:
i corrected that in my project, I hope that I understood correctly.